doc/getting_started/index.rst

```diff
@@ -3,319 +3,372 @@
+              make gcc gcc-multilib
 
-Install west
-============
+      #. Install needed Python packages:
 
-First, install ``west`` using ``pip3``:
+         .. code-block:: bash
 
-.. code-block:: console
+            pip3 install --user -U west
+            pip3 install --user -r https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/master/scripts/requirements.txt
+            echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
+            source ~/.bashrc
 
-   # Linux
-   pip3 install --user -U west
+         The ``--user`` option puts installed packages into your
+         ``~/.local/bin folder``.  It's very important you permanently update the
+         PATH environment variable with the location of these packages (in your
+         .bashrc profile, as shown).
 
-   # macOS (Terminal) and Windows (cmd.exe)
-   pip3 install -U west
+         |p|
 
-See :ref:`west-install` for additional details on installing west.
+      #. Verify the version of cmake installed on your system using ``cmake
+         --version``, and if it's not version 3.13.1 or higher, download
+         and update cmake:
 
-.. _clone-zephyr:
+         .. code-block:: bash
+
+            cd ~
+            wget https://github.com/Kitware/CMake/releases/download/v3.15.3/cmake-3.15.3-Linux-x86_64.sh
+            chmod +x cmake-3.15.3-Linux-x86_64.sh
+            sudo ./cmake-3.15.3-Linux-x86_64.sh --skip-license --prefix=/usr/local
+            hash -r
+
+   .. group-tab:: macOS
+
+      #. On macOS, install :program:`Homebrew` by following instructions on the `Homebrew
+         site`_, and as shown here. Homebrew is a free and open-source package management system that
+         simplifies installing software on macOS.  While installing Homebrew,
+         you may be prompted to install additional missing dependencies; please follow
+         any such instructions as well.
+
+         .. code-block:: bash
+
+            /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+      #. Then, install these host dependencies with the ``brew`` command:
+
+         .. code-block:: bash
+
+            brew install cmake ninja gperf ccache dfu-util qemu dtc python3
+
+      #. Install needed python packages with these commands:
+
+         .. code-block:: bash
+
+            pip3 install -r https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/master/scripts/requirements.txt
+
+      .. _Homebrew site: https://brew.sh/
+
+   .. group-tab:: Windows
+
+      .. note:: If you're running Windows 10 (version 1809 or later)
+         you can run Ubuntu binaries natively within the built-in Windows
+         Subsystem for Linux (WSL). Using WSL will also let you to use
+         the Zephyr SDK. `Install the Windows Subsystem for Linux (WSL)
+         <https://msdn.microsoft.com/en-us/commandline/wsl/install_guide>`__, and
+         then follow the Linux (Ubuntu) Getting Started instructions.
```

Comment:
I suspect the answer is "it depends". As a coincidence I'm using a USB-serial adapter from WSL right now and it Just Works.

Reply:
I think it is OK to have it but I would add a warning that the recommended option is to use `cmd.exe`

Reply:
I don't use Windows often but when I do I always use PowerShell. It's not perfect (nothing is) but it's orders of magnitude more usable than `cmd.exe`... Any particular reason for keeping Windows users stuck in the prehistory of computing? Because Windows users don't care, otherwise they'd be using Linux? :-P

Reply:
I followed the instructions as written for Linux/Ubuntu on WSL and made it all the way through without a hitch, until getting to "west flash" where it failed both using  just using "west flash" or using sudo:
```
~/zephyrproject/zephyr$ west flash
-- west flash: rebuilding
ninja: no work to do.
-- west flash: using runner pyocd
-- runners.pyocd: Flashing file: /home/dbkinder/zephyrproject/zephyr/build/zephyr/zephyr.hex
No connected debug probes
0014885:CRITICAL:__main__:uncaught exception:
Traceback (most recent call last):
  File "/home/dbkinder/.local/lib/python3.6/site-packages/pyocd/__main__.py", line 338, in run
    self._COMMANDS[self._args.cmd](self)
  File "/home/dbkinder/.local/lib/python3.6/site-packages/pyocd/__main__.py", line 468, in do_flash
    with session:
  File "/home/dbkinder/.local/lib/python3.6/site-packages/pyocd/core/session.py", line 284, in __enter__
    assert self._probe is not None
AssertionError
ERROR: command exited with status 1: pyocd flash -e sector -t nrf52 -f 4000000 /home/dbkinder/zephyrproject/zephyr/build/zephyr/zephyr.hex
```

@MaureenHelm comments on the [dev mailing list](https://lists.zephyrproject.org/g/users/topic/31871754#1496) that you "unfortunately cannot use pyocd in the Windows 10 Linux subsystem. WSL has limited support for USB", so we may be stuck. You can go so far using WSL, then need to switch to native windows tools for flashing/debugging?  That sounds like a messy solution vs. just using board/architecture-specific native windows tools (and not using the Zephyr SDK).

Reply:
> @marc-hb Have you flashed a board from WSL? We should talk.

No, I was referring to another, (simpler?) and working USB device. This device doesn't require using `root` BTW.

I suggest restricting mentions of WSL to the "advanced" page(s) with a short caveat that pyocd doesn't work last time someone tried (WSL is evolving very fast and running an actual Linux kernel "soon")